WVAanalysisRun2/src/setOutputTree.cc

```diff
@@ -19,6 +19,7 @@ void setOutputTree::initializeVariables()
   isBoostedSignal=-999;
   wSampleWeight=-999;
   genWeight=1;
+  originalWeight=-1;
```

Comment:
Also need to reset the AQGC vector, so add this line below:
AQGCweights.clear();